rs/sns/testing/README.md

```diff
@@ -1,8 +1,49 @@
 # SNS testing
 
+## Prerequisites
+
+SNS testing CLI uses identities managed by `dfx`. Currently, the scenario requires two identities:
+the first one represents developer identity and is used to submit NNS/SNS proposals, and the second one
```

Comment:
I wonder if we need to expose the second one to the user. Could we just mint tokens on-demand?

Reply:
I had the same thought earlier today, and I already removed the necessity to request it from the user: https://github.com/serokell/ic/pull/6